lib/core/src/Cardano/Pool/DB/Sqlite.hs

```diff
@@ -515,6 +505,7 @@ newDBLayer trace fp timeInterpreter = do
             deleteWhere [ PoolRegistrationPoolId ==. pool ]
             deleteWhere [ PoolRetirementPoolId ==. pool ]
             deleteWhere [ StakeDistributionPoolId ==. pool ]
+            deleteWhere [ DelistedPoolId ==. pool ]
```

Comment:
Same as above.

Reply:
Fixed in 12f5426917b675d1c9ee01fc2f48510d95ecf422.